sparkle/src/components/Dropdown.tsx

```diff
@@ -35,9 +35,9 @@ export const menuStyleClasses = {
         variant: {
           default: cn(
             "s-p-2",
+            "hover:s-bg-blue hover:s-bg-muted-night",
```

Comment:
why blue? and I think it's broken? 

<img width="476" height="310" alt="Image" src="https://github.com/user-attachments/assets/a333002f-f80c-48b5-9fbd-324c716be64f" />